e2e_tests/provider_cfg/tpm/config.toml

```diff
@@ -16,4 +16,4 @@ manager_type = "OnDisk"
 provider_type = "Tpm"
 key_info_manager = "on-disk-manager"
 tcti = "mssim"
-owner_hierarchy_auth = "tpm_pass"
+owner_hierarchy_auth = "hex:74706d5f70617373" # "tpm_pass" in hex
```

Comment:
Is that different from `"str:tpm_pass"`?

Reply:
No, it isn't, just wanted to test it.